app/services/encounter_service.rb

```diff
@@ -51,6 +51,14 @@ def find_encounter(type:, patient:, encounter_datetime:, provider:, program:)
              .first
   end
 
+  def destroy_lab_order_lims(encounter_id)
```

Comment:
Handle this in the encounter model under after void. It already voids all orders. You might want to segregate this code to only handle lab orders only